paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -805,6 +806,12 @@ internal abstract class BaseSheetViewModel(
 
     abstract fun onUserCancel()
 
+    internal fun closeScreens() {
```

Comment:
This is what I mentioned in another comment. I wonder if we can use `newCoroutineContext` with `viewModelScope` to remove the need for this (in a follow up).